libs/sdk-ui-kit/src/SettingItem/SettingItem.tsx

```diff
@@ -8,7 +8,9 @@ import { ActionType, ISettingItem } from "./typings";
 import { Button } from "../Button";
 import { Bubble, BubbleHoverTrigger } from "../Bubble";
 
-const RIGHT_TOOLTIP_BUBBLE_ALIGN_POINTS = [{ align: "cr cl" }];
+const DEFAULT_ALIGN_POINT_TITLE_TOOLTIP = [{ align: "cr cl" }, { align: "bl tl" }, { align: "bc tc" }];
+
+const DEFAULT_ALIGN_POINT_ACTION_TOOLTIP = [{ align: "cl cr" }, { align: "bc tc" }];
```

Comment:
Just a confirmation, should we change `{ align: "cl cr" }` → `{ align: "cr cl" }`?

Reply:
Yes, the tooltip of the Action Button should be `cl cr`

Reply:
ok then